tests/functional-swap.rs

```diff
@@ -168,10 +168,6 @@ async fn run_swap(
     // run until bob has the btc funding address
     let address = retry_until_bitcoin_funding_address(cli_bob_progress_args.clone());
 
-    // this seems to improve reliability, probably there is a race condition somewhere with the address being funded too early
-    // FIXME: change this once https://github.com/farcaster-project/farcaster-node/issues/225 is fixed
-    thread::sleep(time::Duration::from_secs(10));
-
     // fund the bitcoin address
     let amount = bitcoin::Amount::ONE_SAT * 100000150;
```

Comment:
bitcoin::Amount::from_sat(100000150)

Reply:
Will re-factor all the amounts/funding once the progress messages are in place properly.